src/main/java/io/neonbee/internal/verticle/WatchVerticle.java

```diff
@@ -154,7 +156,13 @@ public WatchVerticle(Path watchPath, long interval, TimeUnit unit, boolean paral
     public void start(Promise<Void> startPromise) {
         Vertx vertx = getVertx();
         if (NeonBee.get(vertx).getOptions().doNotWatchFiles()) {
```

Comment:
In my opionion the we shouldn't check in WatchVerticle if the NeonBee option "doNotWatchFiles" ist set or not. It is a NeonBeeOption and should only change the behavior of NeonBee. If I want to deploy a Module, which contains a WatchVerticle I don't want that this NeonBee option prevents it.

We have 2 verticles that extends the WatchVerticle DeployerVerticle and ModelRefresher. If NeonBee option "doNotWatchFiles" is set, we simply shouldn't deploy this verticles.

Reply:
I see your point, however then we have a different behaviour in `JobVerticle` than we have in `WatchVerticle`. And if you don't want to have _any_ files watched, that's what the NeonBee option is for. The question is: Should we "enforce" it, then I would say, adding it to the `WatchVerticle` is kind of how we did it in the `JobVerticle` as well, or should we _not_ enforce it, then I would agree to your option. What do you think?

Reply:
Maybe its the name of the option. I thought we introduced the option "doNotWatchFiles" with the idea to stop NeonBee watching the folders "models" and "verticles". Or did we had another idea in mind?

If this was the idea, I wouldn't enforce it. I got your point with the similarity to "shouldDisableJobScheduling", but then maybe we simply should rename the option into "doNotWatchModules" and try to "remove" the models folder as soon as possible. Or rename it to "doNotWatchResources" or "doNotWatchModelsAndModules".

To be honest in my opinion "shouldDisableJobScheduling" is also not really required. I'd prefer disabling the "unwanted" JobVerticles via their config.

Reply:
Well both options are particularly useful in tests, because in many / most tests you don't want to have files watch nor you want job verticles, Thus the default option for tests has been set to "true / true" so no jobs are started and no files will be watched. Maybe we should move this debate to an issue instead to have it in our backlog and resolve this conversation?

Reply:
I don't know, because this change then introduces a behavior which maybe is unwanted and needs to be removed afterwards. On the other hand this PR is so big in the meanwhile, that it is hard to review. So maybe it makes sense to merge it now as it is and do the "removal" maybe later.

I like none of these options, so you can decide :)

Reply:
I would like to merge this change *now* :D if you permit! 👯

Then this beast and the whole chapter on non-running tests is hopefully and FINALLY closed!